### PR TITLE
Can swipe to switch betweens query results

### PR DESCRIPTION
**Pull request by ger-benjamin**
Ref to and require to close https://github.com/camptocamp/c2cgeoportal/issues/1800

Example on: https://ger-benjamin.github.io/ngeo/master/examples/contribs/gmf/apps/mobile/index.html (swipe deactivated for computers mouse)

This behavior is required by GMF v2.**0** but my code is not optimal, because:
 - I think we don't need "touch" on desktop and I require this library on all gmf components.
 - Angular-Touch support only right-to-left and left-to-right gestures. I'm sur we will need someday a swipe-up or swipe-down support...

I don't know if someone has a better idea to support gestures for GMF 2.0. Perhaps we can use this as is for now and make a better architecture to choose mobile/desktop requierments and to find a better library to support gestures for the v2.1.

---

~same discussion on: #550

### CODE REVIEW

**Comment by ger-benjamin** [comment]
~same discution on: https://github.com/camptocamp/ngeo/pull/550

File: contribs/gmf/examples/simple.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+!DOCTYPE html>
```
Comment (sbrunner) [inline contribs/gmf/examples/simple.html]: typo

**Comment by sbrunner** [comment]
Looks good for me :-)

**Comment by ger-benjamin** [comment]
I want to wait on the approval and/or comments of @pgiraud or @fgravin (about Angular-Touch)

**Comment by fgravin** [comment]
LGTM

**Comment by fgravin** [comment]
angular-touch is from main repo, i have blind trust.

**Pull request merged**